.github/workflows/delete-review-app.yml

```
@@ -145,12 +145,7 @@
         uses: ./.github/actions/delete-control-plane-app
         with:
           app_name: ${{ env.APP_NAME }}
-          org: ${{ env.CPLN_ORG }}
-          github_token: ${{ secrets.GITHUB_TOKEN }}
-        env:
-          APP_NAME: ${{ env.APP_NAME }}
-          CPLN_ORG: ${{ secrets.CPLN_ORG }}
-          CPLN_TOKEN: ${{ secrets.CPLN_TOKEN }}
+          org: ${{ vars.CPLN_ORG_STAGING }}
```

actionlint (1.7.4)

input "org" is not defined in action "Delete Control Plane App" defined at "./.github/actions/delete-control-plane-app". available inputs are "app_name", "cpln_org"

(action)